crates/bitcoin/src/parser.rs

```diff
@@ -674,6 +677,13 @@ pub(crate) mod tests {
b8b18d14c453d11e9bf7fd37722438642fb6a232c678d951b3491c2389ab25a72adeefdf9127e1bdf2ff67fd126273a451920dee6d879fa75c6956392f7317a75cccae4cfc22fc0000e17800000030f40001365f302adef1c26b2299710e11d63dd3486a642fc1c11c5743b1d9b182716acf9dbd70c7e889256214a401990effed277e43005cbf9c13dd90f2a6632ff00132200a8f652ebe8479e0a6c527c1a3d0bca73147341bd4f8c8e1c6dbcb74cf32dcc2707728dc225a7246cf5a736ab7ac548abae74d99d0e50785b924ac6d58de0c24cc2b1ebc77b1cd7355de47fee1667563d7c185630035ad9ada8c88000000000000004d080200004a03814485e0512a43bca96b67e166c6cf7c8da1a1f1e23b17bbbbeb26a58f8e5847a17b8c4e252712669ca7a8fe3937ab53c3f82272996e5ebc898e476d239147bc7e73cc7b2049548503e8d8d75de805480000134000b517263ad4502559cff67fedb1960876a940bc8b49d1179750f135600baebd1ecbe3e3cfa74fbc76447ef7c9ae9e48871dbd4cb6aeb0673e7f2bf88b850556378372e2b6eddaeeee45f3d1070facf78f15f2c542dbeebcb5c4d92c16eb75d6a4c562ffe00879831126ea90701d5b38f9c56d34105f5413a3a26ac77807f96dfb07fefb078f742105c4000005f000002908000b9bc1f94282725a391e870a46e823d22e285cc48a9c446607bb61b3bd6ef2196b22995b2da06ee6fb5ab5ddfb00075e0a50c001be995e004aa3cf05362946911a1794e628e6834a1881f5462870d9c8fe2d8e06f36ca5d55b224c1ff82716e85433a262b75d383f1db2a6ff0d12ffb88cc52fc1315e9039eaea9caceee3d5af4200c2d2c0fc0df756c05bd9f7a1f8690126b130942a3f2aff743747bc8cf6d7c156e6602cb5cb022e2ed4d6b806b7aef5d1530573c34fe3f4c15de99a3796c548e42b216c0214684a91f00b1dcd08d6b46c5dd5e3fdd0dd1d58c04b9ace328c883d942e9fce3408da65430146599317add2f30c57f63bd6aa6c4e14015b7159a388d61311a00063a8006f40153a31a14d0b0c6c1496629aed72b3e60d4d0802fc3ea6237a19d6d9511bff922544a1dafc133167ab3b20552e5145f793e78627abf52581c317f9ba35d852db6ad3f43b3d1c91fde7503ac815b81662bb4bf1bba863b07d015c96b8dbcc25c6a422f9f2d7fcb2a43d824b6fdc6a41d33e8dc7a02164010643803292160054ac30e09d800d6aefd380129442c7ba3c597e535a813500519d92b4cc083000006a492802df387d7f43df4fee807a6a03f96639e0f83d813e9fca9e4d38c9000a52e02829e7e0ddd3e7a1df77d72560fe9faaf59a39e2e5a022d70985bd8c964a6d125e10c1112000000000000000077831a57dec6381c1c11c24000111f2990193d9c1ac14644000003614644000000e4557342e107658cf77bcb7750c760f9fde89cf7198e135914cb7fe65d51bd40029396c47c482e80f8d0f7d667d0add838b6bf53bec000a3ba0e76fd0c613f2002e5fce09eec8f86d4ad4868167bb7996eb903d83c23c592475b06a60624a717fc8b5047dae2b010b960ea3615ab0aec5c468bdd78f3a9a354d1aa68d5346a9a354d1aa674140a32270106c8dc00000000000000eec1a0002ccfa00069f0f8584563f4889996c2dc8d8292d2b1e6630d10f61484cd7fe1aa07c5bc530f7a4bc64b901d7f614d04fa0408ac1eab3966f42f6a38432b00568b4c992694c6e678e2f8e27479de3d223da8bf8463da3413fb89bc6ed2d4d3cf946005c8005628cb12714b004a053802684b97613ef1e4ca29aebcad03ec9af6725880f8d0f7d425dd054fa15bb0721c4b537fcc8587ba12a50dd19063ed003d63769447d813ee7442bd988ad8b7effcc7d4ab99b404ee85ce39360be4633c000edb078c03aa00002340006084e7f3e492fdc7cb4c1dfad6dbcb3fba5283e19344a322577b5f39b33f596464cb6a51a12f2ddc1f8b7d45da48b24a52b59ebbaf5c1464889ac377bc11cfe4a4f9b57f639676c6897d9b5e08000006821c04b295f7286658781d63ff8444369ad9fc4aa20018637f9e5d433dd729f721f8200000000").unwrap();
```

Comment:
Are you sure we want to inline this? It's very big

Reply:
I think so? Unless you have a problem with it

Reply:
It broke my mobile view :sweat_smile: